www/scripts/codecheckerviewer/SourceComponentManager.js

```diff
@@ -0,0 +1,357 @@
+// -------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+define([
+  'dojo/dom-construct',
+  'dojo/dom-attr',
+  'dojo/dom-class',
+  'dojo/_base/declare',
+  'dojo/data/ItemFileWriteStore',
+  'dojox/grid/DataGrid',
+  'dijit/ConfirmDialog',
+  'dijit/form/Button',
+  'dijit/form/SimpleTextarea',
+  'dijit/form/TextBox',
+  'dijit/layout/BorderContainer',
+  'dijit/layout/ContentPane'],
+function (dom, domAttr, domClass, declare, ItemFileWriteStore, DataGrid,
+  ConfirmDialog, Button, SimpleTextarea, TextBox, BorderContainer,
+  ContentPane) {
+
+  var DeleteComponentDialog = declare(ConfirmDialog, {
+    constructor : function () {
+      this._confirmLabel = new ContentPane({
+        class : 'delete-confirm-text',
+        innerHTML : '<span class="warningHeader">You have selected to ' +
+                    'delete a source component!</span>'
+      });
+    },
+
+    postCreate : function () {
+      this.inherited(arguments);
+      this.addChild(this._confirmLabel);
+    },
+
+    onExecute : function () {
+      CC_SERVICE.removeSourceComponent(this.componentName);
+      this.listSourceComponent.refreshGrid();
+    }
+  });
+
+  var EditSourceComponent = declare(ContentPane, {
+    postCreate : function () {
+      var that = this;
+
+      this._errorMessage = dom.create('div', {class : 'mbox mbox-error hide'});
+      dom.place(this._errorMessage, this.domNode);
+
+      this._componentName = new TextBox({
+        class : 'component-name',
+        placeHolder : 'Name of the source component...'
+      });
+      this._placeFormElement(this._componentName, 'name', 'Name*');
+
+      this._componentValue = new SimpleTextarea({
+        class : 'component-value',
+        rows : 10,
+        placeholder : 'Value of the source component. Each line must start '
```

Comment:
I would recommend a small rephrasing in the placeholder:
```
with a "+" (results from this path should be listed) or "-" (results from this path should not be listed) sign. E.g.: '+/a/b/x.cpp or -/a/b/'
```
I would do this rephrasing in the command line and in the user guide too.